obj.go

```diff
@@ -54,13 +55,23 @@ func (cc *Conn) AddObj(o Obj) Obj {
 }
 
 // GetObj gets the specified Obj without resetting it.
-func (cc *Conn) GetObj(o Obj) ([]Obj, error) {
```

Comment:
Can you leave the old API untouched and instead introduce a new function to avoid breaking the API? Perhaps `GetObject`, or `GetSingleObject`